tools/extensions/generate_extension_db.py

```diff
@@ -81,7 +97,13 @@ def GetExtensionMetadata(target):
 
 
 if __name__ == '__main__':
-  output_path = sys.argv[1]
+  try:
+    output_path = (os.environ["EXTENSION_DB_PATH"]
```

Comment:
`output_path = os.getenv('EXTENSION_DB_PATH', sys.argv[1])`

Reply:
this would break if `sys.argv[1]` is not set - hence the `if ... else`

i think this code could be a bit prettier with different formatting but i think the logic is correct

Reply:
i updated to `output_path = os.getenv("EXTENSION_DB_PATH") or sys.argv[1]` which is more concise and works as expected